assay/skills/backend-quality/references/scoring-model.md

```
@@ -0,0 +1,86 @@
+# Scoring Model
+
+How findings are aggregated into a verdict. The plugin produces standalone verdicts (no workflow concepts). Workflow tools map plugin verdicts to their own status values.
+
+## Plugin Verdict
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)